h/static/scripts/streamer.js

```diff
@@ -27,33 +20,18 @@ var socket;
  * @return The push notification service client.
```

Comment:
The `@return` line can be removed here since this was documentation for the `connect()` function